test/parallel/test-dgram-createSocket-type.js

```diff
@@ -38,3 +38,22 @@ validTypes.forEach((validType) => {
     socket.close();
   });
 });
+
+// Ensure buffer sizes can be set
+{
+  const socket = dgram.createSocket({ type: 'udp4',
```

Comment:
I'm a little surprised the linter doesn't complain about this. Can you move `type` to the next line with two spaces indentation.